src/app/products/web3storage/page.tsx

```diff
@@ -0,0 +1,78 @@
+import { Header } from "@/components/nav"
+import { Metadata } from "next"
+import Image from "next/image"
+import Ring from '@/../public/images/illustrations/ring.png'
+
+export const metadata: Metadata = {
+  title: 'Features'
+}
+
+export default async function FeaturesPage () {
+  return (
+    <div>
+      <div className="bg-grad">
+        <Header />
+        <section className='max-w-6xl mx-auto pt-20 md:pt-28 px-3 pb-8'>
+          <div className="md:flex flex-row-reverse">
+            <div className="md:w-2/3">
+              <h1 className="font-bold text-4xl text-zinc-950">What is web3.storage</h1>
+              <p className="leading-relaxed pt-8 max-w-3xl text-xl text-zinc-950">At the core of web3.storage is a storage service to safely secure and make your data available - giving developers the power of decentralized storage and content addressing via simple client libraries or an HTTP API.</p>
+            </div>
+            <div className="hidden md:block w-1/3">
+              <Image src={Ring} alt='Ring' className="rotate-45" />
+            </div>
+          </div>
+        </section>
+      </div>
+      <section className="lg:-mt-36 bg-zinc-950 text-zinc-200">
+        <div className='max-w-6xl mx-auto py-24 px-3 md:flex flex-row-reverse items-end'>
+          <div className="md:w-2/3">
+            <p className="text-md leading-relaxed">
+              Content uploaded to web3.storage is stored on our instance of <a href="https://github.com/elastic-ipfs/elastic-ipfs">Elastic IPFS</a>, a cloud-native implementation of IPFS that our team wrote to stay reliable and performant with the scale of uploads we were seeing. Elastic IPFS ensures your data is continuously and quickly available over the IPFS network once your data is uploaded.
```

Comment:
ok I see a bunch is outdated, are we tracking somewhere what still needs an update in copy?

Reply:
Not yet. In general the copy on the website is too opaque and marketing heavy. That's kinda fine for the the call-to-action of a landing page, but after that I think the rest of the copy should borrow from our specs and diagrams rather than trying to simplify it. 

One of our strengths is the quality and clarity (and specs) behind what we have built. 

If a user comes to our site looking for a easy way to get a CID for the there data, then we have that covered, boom, done, we don't need to provide reams of vague promises.

If a technical user comes looking to understand why they should use web3.storage, we have a ton of good explainers, but none are present here yet, and I don't think the site in it's current form does much to win them over. too many vague promises, not enough concrete explanation. Much of the burden of that falls on the docs site, but I think the features page here should also be a lot clearer. I can take a pass at that next week.

cc  @heyjay44